images/kubekins-e2e/variants.yaml

```diff
@@ -1,13 +1,13 @@
 variants:
   experimental:
     CONFIG: experimental
-    GO_VERSION: 1.12.12
+    GO_VERSION: 1.13.4
     K8S_RELEASE: stable
     BAZEL_VERSION: 0.28.1
     UPGRADE_DOCKER: 'true'
   master:
     CONFIG: master
-    GO_VERSION: 1.12.12
+    GO_VERSION: 1.13.4
```

Comment:
can you also do ./../krte/ ? (KIND Runtime EnvironmenT) 🙃

Reply:
done, I think